pipelines/batch/src/main/java/com/google/fhir/analytics/HapiRowDescriptor.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020-2023 Google LLC
+ * Copyright 2020-2024 Google LLC
```

Comment:
**question:** Did you do this? lol

Reply:
Probably automatic from their formatter.